examples/patterns/card/overlay.html

```diff
@@ -6,7 +6,7 @@
 
 <section class="p-strip--image is-light" style="background-image:url('https://assets.ubuntu.com/v1/0a98afcd-screenshot_desktop.jpg')">
   <div class="row">
-    <div class="col-12 prefix-6">
+    <div class="col-6 col-start-large-6">
```

Comment:
I think that the "col-start" class could be better named. Perhaps make it a little more like BEM?

I can't think of an ideal solution now but something like `col-start-6--large` might be more in-line with the rest of Vanillia.